README.md

```diff
@@ -110,6 +110,18 @@ pip install enum34
 
 The rest of the build/install steps are the same, save for changing out the version numbers from PyQt4 to PyQt5.
 
+### Dependencies:
+To enable proper networking:
+
+```
+pip install msgpack
+```
+
+To enable Posix C++ demangling:
+
+```
+pip install cxxfilter
```

Comment:
So when I do a `pip search cxxfilter` I don't find it. But I do find it when doing `pip search cxxfilt`